PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/run/SparkBatchRemoteRunState.kt

```diff
@@ -63,15 +67,16 @@ open class SparkBatchRemoteRunState(val serverlessSparkSubmitModel: SparkSubmitM
                             MessageInfoType.Hyperlink ->
                                 BrowserUtil.browse(URI.create(messageWithType.value))
                             else ->
+                            {
                                 consoleView!!.print("ERROR: ${messageWithType.value}\n", ConsoleViewContentType.ERROR_OUTPUT)
+                                ClassifiedException(messageWithType.value).logStackTrace()
+                            }
                         }
                     },
                     { err ->
-                        val errWriter = StringWriter()
-                        err.printStackTrace(PrintWriter(errWriter))
-
-                        val errMessage = Optional.ofNullable(err.message)
-                                .orElse(err.toString()) + "\n stack trace: " + errWriter.buffer.toString()
+                        val classifiedExp = ClassifiedException(err)
+                        val errMessage = classifiedExp.message
```

Comment:
`errMessage` is only used once below, I think we can remove the definition and just use `classifiedExp.message` when needed.

Reply:
the following event and console view will use this variable

---
In reply to: [225444567](https://github.com/Microsoft/azure-tools-for-java/pull/2100#discussion_r225444567) [](ancestors = 225444567)